source/guide_wallabag.rst

```diff
@@ -0,0 +1,147 @@
+.. highlight:: console
+
+########
+Wallabag
+########
+
+Wallabag_ is a read later solution like `Firefox Pocket`_ to save and organize articles between devices and make them offline available. This is the server-side application, it will fetch articles and save the content and images on the server when a link is provided. There is also client software for browsers and mobile devices available which can be used to download the fetched articles and add new articles to the server.
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of
+
+  * PHP_
+  * MySQL_
+  * domains_
+
+Prerequisites
+=============
+
+We're using PHP_ in the stable version 7.1:
+
+::
+
+ [isabell@stardust ~]$ uberspace tools version show php
+ Using 'PHP' version: '7.1'
+ [isabell@stardust ~]$
+
+.. include:: includes/my-print-defaults.rst
+
+Also, the domain you want to use for Lychee must be set up as well:
+
+.. include:: includes/web-domain-list.rst
+
+Installation & Configuration
+==============================
+
+First get the Wallabag source code from GitHub_:
+
+::
+
+  [isabell@stardust ~]$ git clone https://github.com/wallabag/wallabag.git ~/html
+  Cloning into '/home/isabell/wallabag'...
+  remote: Counting objects: 46655, done.
+  remote: Compressing objects: 100% (23/23), done.
+  remote: Total 46655 (delta 10), reused 20 (delta 9), pack-reused 46620
+  Receiving objects: 100% (46655/46655), 58.15 MiB | 20.81 MiB/s, done.
+  Resolving deltas: 100% (26623/26623), done.
+  [isabell@stardust ~]$
+
+Change to that folder and run ``make install``. During the installation process, you will be asked for quite a lot configuration settings. Bu you only need to set up the following informations:
```

Comment:
'quite a lot configuration settings' -> 'a lot of configuration settings'. 'information' should be singular.

Reply:
Oh, and 'Bu' -> 'But'.

Reply:
fixed